ipkipkorir/SendITParcel
# #161916354 Implementing UI feedback

**What does this PR do?**
Move UI pages into UI folder

**Description of tasks accomplished**
- Create folder named UI within the SendITParcel repository
- Move UI files into the UI folder

**How should this be manually tested?**
- Clone the repository: https://github.com/ipkipkorir/SendITParcel.git 
- Navigate to the UI folder
- Open in a browser the index.html file
- Navigate the links to see the project feature pages

**What are the relevant Pivotal tracker stories?**
#161916354
